public/orbs-public-beta-own-infra.md

```diff
@@ -0,0 +1,74 @@
+# Running Orbs v2 on your own infrastructure
+
+## Prerequisites
+
+* A Linux machine with Docker Swarm
```

Comment:
in the beta page there are some prerequisites which are valid here too - such as, for example, the orbs node address and private key. 

In addition, the section about the Orbs Node Address is relevant here too - describing the usage of the orbs node address and how to keep it with Eth balance in the range 0.5-1

maybe there are other sections we can copy as is?